src/index.js

```diff
@@ -5,7 +5,14 @@ const CALLBACK_ERROR_ARGUMENT = Symbol('ERROR_CALLBACK_ARGUMENT');
 
 export { WAIT_FOR_ACTION, ERROR_ACTION, CALLBACK_ARGUMENT, CALLBACK_ERROR_ARGUMENT };
 
-const fsaCompliantArgumentCb = action => action.payload || action.data || {};
+const fsaCompliantArgumentCb = action => {
+  let res;
+
+  res = action.payload || action.data;
+  if (res === undefined) res = {};
```

Comment:
This change will allow `null`, `""` and `NaN` to pass. I'm not sure if that would be a problem for other other people, but it would change the functionality of this middleware a little.

Reply:
Also if you do this `data`, why not payload too?  Is payload always suppose to be an object for FSA?

Reply:
Something data = 0 and this is meaningful value. Similar and "".

Reply:
Whether an object should always be returned?

Reply:
As [flux-standard-action](https://github.com/acdlite/flux-standard-action) says:

> ### `payload`
> **The optional payload property MAY be any type of value.** It represents the payload of the 
action. Any information about the action that is not the type or status of the action should be part of the payload field.
>
> By convention, if error is true, the payload SHOULD be an error object. This is akin to rejecting a promise with an error object.
> 
> ### `error`
> 
> The optional error property MAY be set to true if the action represents an error.
> 
> An action whose error is true is analogous to a rejected Promise. By convention, the payload SHOULD be an error object.
> 
> If error has any other value besides true, including undefined and null, the action MUST NOT be interpreted as an error.

In an FSA, the `payload` can also be a zero number.

More than that, it seems that I mistook the value of `error` property as an error object but in fact it is a boolean. When this boolean is set to `true`, the `payload` is the error object. Maybe we should change more in the next major release.

Reply:
And now how can i write my code, that dont break in future, but my need use zero-numer as data or payload?

Reply:
I am not using FSA, so I don't have an opinion on this, I was just pointing out that this PR was an API change.  @kolesoffac you could use `CALLBACK_ARGUMENT` to override this in the current version to suit your needs if you need a quick solution: https://github.com/Chion82/redux-wait-for-action#overriding-the-default-promise-arguments

Reply:
Many thanks, i resolve this problem use CALLBACK_ARGUMENT.